modules/swagger-codegen/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -14,7 +14,12 @@ import { Response, ResponseContentType }                     from '@angular/http
 import { CustomQueryEncoderHelper }                          from '../encoder';
 {{/useHttpClient}}
 
+{{^useRxJS6}}
 import { Observable }                                        from 'rxjs/Observable';
+{{/useRxJS6}}
+{{#useRxJS6}}
+import { Observable }                                    from 'rxjs';
+{{/useRxJS6}}
 {{^useHttpClient}}
 import '../rxjs-operators';
```

Comment:
the `import '../rxjs-operators';` should be removed, as pipeable operators are already available as of RxJS 5.5